packages/react-scripts/template/src/service-worker-registration.js

```diff
@@ -0,0 +1,38 @@
+export function register() {
+  if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
```

Comment:
Without an accompanying unit test (which would likely require some heavy-duty mocking), the code including and past this line has a negative effect on my project's coverage report. I left a comment on an upstream issue (https://github.com/facebookincubator/create-react-app/issues/1299#issuecomment-282539816) regarding having the ability for Jest to ignore files such as this.